CONTRIBUTING.md

```diff
@@ -75,7 +75,7 @@ $ git rebase -i HEAD~3
 If your branch conflicts with the master branch, you will need to rebase and repush it with the following commands:
 
 ``` bash
-$ git remote add upstream git@github.com:php-http/repo-name.git
+$ git remote add upstream git@github.com:orga/repo-name.git
```

Comment:
Low prio IMO, moved to documentation#44. For now it is just easier to copy files from boilerplate without thinking. :stuck_out_tongue: